ecpp

```diff
@@ -112,7 +140,102 @@ def main():
+    with open(str(moduli)) as f:
+        inputfile = filter(None, (line.rstrip() for line in f))
+        count_line = 1
+        for line in inputfile:
+            if not line.startswith('#'):
+                prime_moduli = (line.split(" ")[6])
+                int_prime_moduli = int(prime_moduli, 16)
+                status = False
+                print("=====================================================")
+                print("Searching cert for: " + str(int_prime_moduli))
+                print("Moduli prime on line no: " + str(count_line))
+                for cert in cert_files:
+                    if cert.endswith('.out'):
+                        path = cert
+                        cert_candidate = candidate_finder(path)
+                        if cert_candidate:
+                            cert_candidate = int(cert_candidate, 16)
+                            if int_prime_moduli == cert_candidate:
+                                print("[+] cert file " + path + " found for moduli prime on line: " + str(count_line))
+                                status = True
+                                if verify_opt == True:
+                                    verify(path)
+                if status == False:
+                    print("[-] cert file not found for moduli prime")
+                    if primo_opt:
+                        print("[!] creating primo input file in " + primo_in)
+                        converter(prime_moduli)
+            count_line = count_line + 1
+
+def usage():
+    print("usage here: ")
+
+def main():
+    moduli = None
+    moduli_opt = False
+    certs = None
+    certs_opt = False
+    verify = None
+    verify_opt = False
+    primo_opt = False
+
+    try:
+        opts, args = getopt.getopt(sys.argv[1:], 'm:c:pvh', ['moduli=', 'certs=', 'primo=', 'verify=', 'help'])
+    except getopt.GetoptError:
+        print("[!] parsing of command line args failed\n")
+        usage()
+        sys.exit(2)
+
+    for opt, arg in opts:
+        if opt in ('-h', '--help'):
+            usage()
+            sys.exit(2)
+        elif opt in ('-m', '--moduli'):
+            moduli_opt = True
+            moduli = arg
+        elif opt in ('-c', '--certs'):
+            certs_opt = True
+            certs = arg
+        elif opt in ('-p', '--primo'):
+            primo_opt = True
+        elif opt in ('-v', '--verify'):
+            verify_opt = True
+        else:
+            usage()
+            sys.exit(2)    
+
+    if not moduli:
+        print("[!] moduli file not specified\n")
+        usage()
+        sys.exit(2)
+
+    if not certs:
+        print("[!] certs directory not specified\n")
+        usage()
+        sys.exit(2)
+    
+    if moduli_opt and not certs_opt and not verify_opt:
+        correlate(moduli)
+    elif moduli_opt and certs_opt and not verify_opt:    
+        correlate(moduli, certs)
+    elif moduli_opt and certs_opt and verify_opt:    
+        correlate(moduli, certs, verify_opt)
```

Comment:
why not use just this line? the other don't do anything different given how parameters are defined...